### DISCUSSION TITLE
Fails to build ruby 3.1.0 (macOS 12.1, M1 Pro)

### DISCUSSION THREAD

**Discussion by josh-long-raylo** [rbenv/ruby-build#1906] [Build failures]
I'm unable to install ruby 3.1.0 on **MacBook Pro (16-inch, 2021), Chip Apple M1 Pro macOS 12.1**

When attempting `rbenv install 3.1.0` I get an error `mach-o file, but is an incompatible architecture (have 'arm64', need 'arm64e'`.
I've attached the full log below.
[ruby-build.20220112153258.3838.log](https://github.com/rbenv/ruby-build/files/7855345/ruby-build.20220112153258.3838.log)

It might be worth noting, ruby 3.0.3 installs fine

Any help would be greatly appreciated - thanks :)

**Comment by GithubSupport6** [comment]
I have the same issue, did you find a solution?
